src/main/kotlin/lotto/model/NumbersGenerator.kt

```diff
@@ -0,0 +1,16 @@
+package lotto.model
+
+fun interface NumbersGenerator {
+    fun generate(): List<Int>
+}
```

Comment:
```suggestion
fun interface LotteriesGenerator {
    fun generate(): List<Lotto>
}
```

기왕 NumbersGenerator 라는 인터페이스를 선언해주신 것처럼,
더 큰 단위로 Lotteries 그 자체를 만드는 인터페이스를 선언해보면 어떨까 제안해봅니다.